pyfolio/tears.py

```diff
@@ -153,13 +154,12 @@ def create_full_tear_sheet(returns,
     if returns.index[0] < benchmark_rets.index[0]:
         returns = returns[returns.index > benchmark_rets.index[0]]
 
-    if slippage is not None and transactions is not None:
+    if (unadjusted_returns is None) and (slippage is not None) and\
```

Comment:
Is there a case where someone would be passing in both adjusted and unadjusted returns?